Scripts/Audio.cs

```diff
@@ -183,4 +183,138 @@ public string output
     }
 
 
+
+    [Serializable]
+    public class SpeechRecognitionOptions
+    {
+        public enum SpeechRecognitionService
+        {
+            Unified,
+            Google,
+            Aws,
+            Deepgram
+        }
+
+        public enum SpeechRecognitionLanguageCode
+        {
+            DE_DE,
+            EN_AU,
+            EN_GB,
+            EN_US,
+            ES_US,
+            FR_CA,
+            FR_FR,
+            IT_IT,
+            JA_JP,
+            KO_KR,
+            PT_BR,
+            ZH_CN
+        }
+
+        public string Service
+        {
+
+            get
+            {
+                switch (_service)
+                {
+                    case SpeechRecognitionService.Unified:
+                        return "unified";
+                    case SpeechRecognitionService.Google:
+                        return "unified:google";
+                    case SpeechRecognitionService.Aws:
+                        return "unified:aws";
+                    case SpeechRecognitionService.Deepgram:
+                        return "unified:deepgram";
+                    default:
+                        Logger.LogError("Unknown audio format");
+                        return null;
+                }
+            }
+        }
+
+        public string Encoding
```

Comment:
I don't think we need `Encoding` as we control the microphone implementation so we know what it's going to be